src/Orleans/Properties/AssemblyInfo.cs

```diff
@@ -26,6 +26,7 @@
 [assembly: InternalsVisibleTo("OrleansProviders")]
 [assembly: InternalsVisibleTo("OrleansHost")]
 [assembly: InternalsVisibleTo("OrleansAzureUtils")]
+[assembly: InternalsVisibleTo("OrleansServiceBus")]
```

Comment:
I totally agree. To avoid this reference, I think there's two solution:
1. short term: maintain a separate timer in EHSP
2. long term: package the utility class which is highly shareable, such as safe timer, to a separate nugget, so we won't wire unnecessary reference